modules/update-preparing-evaluate-alerts.adoc

```diff
@@ -10,4 +10,6 @@ Two alerts fire when an API is in use that will be removed in the next release:
 * `APIRemovedInNextReleaseInUse` - for APIs that will be removed in the next {product-title} release.
 * `APIRemovedInNextEUSReleaseInUse` - for APIs that will be removed in the next {product-title} Extended Update Support (EUS) release.
 
-If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version. You can use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs.
+If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version.
+
+Use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs, because the alerts do not provide this information. Additionally, some APIs might not trigger these alerts but are still captured by `APIRequestCount`. The alerts are tuned to be less sensitive to avoid alerting fatigue in production systems.
```

Comment:
>less sensitive

How so?

Reply:
Is there actually any information available to review in the alert itself? I thought it was pretty much just that hey, this alert is firing, now go use `APIRequestCount` to see what it really is. So if that's the case, then I think we likely don't need to go into details on how to review the alert. @wangke19 Let me know what you think.

Reply:
There is interesting information in the alerts, currently `resource`, `group`, and `version` labels.  But (presumably because of cardinality concerns) no information about the user that needs bumping.  For that, you'd want to dip into `USERNAME,USERAGENT`, as described in [this KCS][1].  I'm agnostic about whether or not it's worth discussing the alerts in detail if we're always going to ask them to dip down into `APIRequestCount` for that user information.

[1]: https://access.redhat.com/articles/6955985

Reply:
If we want to get into reviewing the alerts in more details, let's follow up on that in another PR. Right now this PR is just pulling in some new information to keep the product docs in sync with the kbase

Reply:
@bergerhoffer Agree with you, since the executor is as a user with the cluster-admin role, we don't need to go into details on how to review the alert here.